examples/js/renderers/CanvasRenderer.js

```diff
@@ -7,7 +7,7 @@ THREE.SpriteCanvasMaterial = function ( parameters ) {
 	THREE.Material.call( this );
 
 	this.type = 'SpriteCanvasMaterial';
-
+	this.rotation = 0;
```

Comment:
I wonder if we should note that this property only ensures correct raycasting. It has currently no visual effect on sprites.

Reply:
It used here.
https://github.com/06wj/three.js/blob/e64752d7d9e637351a1ee933072c6d82c60f4dab/examples/js/renderers/CanvasRenderer.js#L585

Reply:
Uh, currently `_context.rotate()` is called with `undefined` as parameter. It's really good to add `rotation` to `SpriteCanvasMaterial` :sweat_smile:

Reply:
Uh oh!